ke/compiler.py

```diff
@@ -109,12 +150,14 @@ def invert_operator(n, expr):
 builtin_operators = {
     "capture": asm.Capture,
     "not": invert_operator,
+    "separate": compile_separate,
```

Comment:
Should we call it `separate`, `separated`, `separator`, or `join`? I like `separated` but I'm leaning towards `join` because that's how the parallel functionality in most programming languages is called

Reply:
I think join wins because its also the shortest, and separate might not be as common to be thought of as join would be.